cloud_governance/policy/helpers/aws/policy/zombie_cluster_operations.py

```diff
@@ -0,0 +1,186 @@
+from cloud_governance.common.logger.init_logger import logger
+from cloud_governance.policy.helpers.aws.aws_policy_operations import AWSPolicyOperations
+
+
+class ZombieClusterOperations(AWSPolicyOperations):
+
+    def __init__(self):
+        super().__init__()
+        self._cluster_prefix = self.config_variable.CLUSTER_PREFIX
+        self._zombie_resource_name = self._environment_variables_dict.get('ZOMBIE_RESOURCE_NAME')
+        self._cluster_ids = []
+        self._zombie_vpc_ids = {}
+
+    def get_zombie_resources(self, cluster_resources: dict, search_global: bool = False) -> dict:
+        """
+        This method filter zombie resource, meaning no active instance for this cluster
+        :param search_global:
+        :param cluster_resources:
+        :return:
+        """
+        if not self._cluster_ids:
+            self._cluster_ids = self._get_global_active_cluster_ids() if search_global else self._get_active_cluster_ids()
+        for cluster_id in self._cluster_ids:
+            if cluster_id in cluster_resources.keys():
+                cluster_resources.pop(cluster_id)
+        return cluster_resources
+
+    def _get_cluster_resources(self, resources_list: list, tags_name: str,
+                               zombie_cluster_id: str = None) -> dict:
+        """
+        This method returns all cluster resources keys that start with cluster prefix
+        :param resources_list:
+        :param tags_name:
+        :return: dictionary of the resources key and id
+        :param resources_list:
+        :return:
+        """
+        result_resources_cluster_id = {}
+        for resource in resources_list:
+            # get resource if cluster_tag is given/ resource_name given/ only cluster prefix given
+            cluster_id = ''
+
```

Comment:
This method returns the dict: {cluster_id: [ resources ] }.

One method to list all cluster resources.